app/controllers/books_controller.rb

```diff
@@ -0,0 +1,6 @@
+class BooksController < ApplicationController
+  def show
+    @book = Book.find(params[:id])
+    @reviews = @book.reviews
+  end
```

Comment:
You generally want to pass as few instance variables to your views as possible. Here's we'd rather see you send `@book` through and in your view call `@book.reviews`.